src/site/content/en/blog/yahoo-japan-news-bfcache/index.md

```diff
@@ -0,0 +1,161 @@
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+When web pages become instant with bfcache, the users tend to stay on page longer, thus increasing the views on ads, leading to the increase of ads revenue.
+
+However, we think there is more to bfcache than just making the page instant.
+
+## bfcache enhances seamless user experience on the website
+
+bfcache is not just about making the pages instant. When the pages are instant, **the website navigation feels more seamless.**
+
+In Yahoo! JAPAN News, one of the major user journey is as follows:
+
+1. Go to the article list
+2. Click on one article to read
+3. Go back to the article list
+4. Click on another article to read
+
+Before bfcache, when users finished reading an article in (step 2), they had to wait for the article list page to load. This could be a friction factor for users who just want to go back to the article list to pick out another article to read.
+
+Another source of friction during backward navigation was the scroll position. In practice, the browser tries to restore the scroll position when a backward navigation happens. However, because of dynamically-added ads or other layout changes, the scroll position often gets incorrectly restored, which could cause users to lose their bearings or even leave the page. This is never an issue when a backward navigation is powered by bfcache: the scroll position is immediately and correctly restored.
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/WlJn1GZnDrSLpEPPHWAn.png", alt="Two filmstrips of a backward navigation from an article to the article listing page. The top is a filmstrip of the process being handled with bfcache which takes 0.3 seconds, whereas the bottom is of the same process being handled without bfcache, which takes 3.3 seconds.", width="800", height="343" %}
+</figure>
+
+Now with bfcache, the friction in the user journey is gone—the users can seamlessly go back to the article list page and pick another article to read instantly.
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/pk7QmKzBA49ADdcJGE7q.gif", alt="An animated image showing the backward navigation flow from an article to the article listing page with and without bfcache. With bfcache, the backward navigation is not only faster, but the scroll position is accurately maintained. Without bfcache, these guarantees cannot be made.", width="800", height="450" %}
+</figure>
+
+In a nutshell, the benefits of using bfcache for Yahoo! JAPAN News includes:
+
+- Increased page view: Users were more likely to navigate within the website when pages were cached with bfcache.
+- Increased revenue: As a result of increased page view per session, the ads impression increased, leading to +9% increase in revenue on mobile compared with the test group without bfcache.
+
+## Conclusion
+
+In short, bfcache not only makes your website instant, but can also reduce friction in overall user experience and increase engagement within your website. 
```

Comment:
Should we add something about Chrome looking at these specific causes?

I guess I'm asking what the takeaway is here, since Chrome _may_ fix both these uses cases automatically (in which case readers may resent having done the work here if that was coming).

WDYT about something like this:

```suggestion
In short, bfcache not only makes your website instant, but can also reduce friction in overall user experience and increase engagement within your website.

The Chrome team is continually looking at bfcache blockers—especially the two reasons listed here as they are common reasons bfcache is not used. In future these may not prevent bfcache usage, but there's no need to wait until then. You can benefit from bfcache by looking at your bfcache blockers and avoiding these common, and other less common, patterns.
```